stock_mts_mto_rule/tests/test_mto_mts_route.py

```diff
@@ -142,6 +142,38 @@ def test_rename_warehouse(self):
         self.warehouse.name = new_warehouse_name
         self.assertEqual(new_rule_name, self.warehouse.mts_mto_rule_id.name)
 
+    def test_standard_mto_mts_buy_route(self):
+        product = self.product
+        mts_mto_route = self.env['stock.location.route'].search([
+            ('name', 'ilike', 'Make To Order + Make To Stock')])
+        buy_route = self.env['stock.location.route'].search([
+            ('name', 'ilike', 'Buy')], limit=1)
+        mto_route = self.env.ref('stock.route_warehouse0_mto')
+        manufacture_route = self.env['stock.location.route'].search([
+            ('name', 'ilike', 'Manufacture')])
+        vendor = self.env['res.partner'].search([], limit=1)
+
+        product.write({'seller_ids': [(6, 0, [vendor.id])]})
+        product.route_ids = [(6, 0, [mto_route.id, manufacture_route.id])]
+
+        bom_bom = self.env.ref('product.product_product_3')
+        bom_material = self.env['mrp.bom'].create({
```

Comment:
The module `stock_mts_mto_rule` doesn't depend of `mrp` anymore

If you run the tests of this module without install another one you will raise an error because mrp.bom model is not defined.

Reply:
removed everything I was editing from mts+mto module